fecgov/openFEC
# Feature/fix missing f4 and f13 committees

## Summary (required)

- Resolves https://github.com/fecgov/openFEC/issues/3382

_Include a summary of proposed changes._

- Added committee_type_map with the correct committee types in  **resources/totals.py** and disabled the index validation since **models/totals.py** is pointing to a **ofec_totals_pacs_parties_vw**.
- Added python test to include inaugural committee (committee_type = X)
- Added a new migration file to drop  **ofec_totals_pacs_vw** and **ofec_totals_parties_vw**  which are no longer needed.


## How to test the changes locally

- checkout the branch : feature/fix-missing-f4-and-f13-committees
- drop/create cfdm_test db
- insert sample data
- your local CMS terminal should point to local api (http://127.0.0.1:5000/)
- bring CMS and API servers up(./manage.py runserver)
- look for an Inaugural committee ( Party - nonqualified) and see if the Financial Summary is getting displayed or not like shown here:

<img width="413" alt="screen shot 2018-10-11 at 3 37 09 pm" src="https://user-images.githubusercontent.com/11650355/46829744-9aad2280-cd6c-11e8-9819-55502a3eedde.png">
<img width="1433" alt="screen shot 2018-10-11 at 3 37 33 pm" src="https://user-images.githubusercontent.com/11650355/46829745-9aad2280-cd6c-11e8-9842-33f2ddbba7cf.png">
<img width="1100" alt="screen shot 2018-10-11 at 3 37 45 pm" src="https://user-images.githubusercontent.com/11650355/46829743-9a148c00-cd6c-11e8-91fd-13a9b15bd1f3.png">




## Impacted areas of the application
List general components of the application that this PR will affect:

-  Financial Summary of Inaugural Committees
